tests/Unit/Wire/AMQPCollectionTest.php

```diff
@@ -613,6 +613,22 @@ public function iterator()
         }
     }
 
+    /**
+     * @test
+     */
+    public function array_access()
```

Comment:
test case names should be meaningful and self explanatory

Reply:
done